### PR TITLE
docs(migration): exclude common configuration to separate section

### PR DESCRIPTION
**Pull request by sumo-drosiek**

### CODE REVIEW

File: docs/migration.md [hunk 1]
```diff
@@ -270,8 +273,48 @@ or [Local Configuration File](#local-configuration-file) for migration details.
 
 This section describes migration steps for Sources managed from the Cloud.
 
+### Common configuration
+
+There is set of configuration common for all of the sources and this secsion is going to cover details of the migration for them.
```
Comment (aboguszewski-sumo) [inline docs/migration.md:278]:
```suggestion
There is set of configuration common for all of the sources and this section is going to cover details of the migration for them.
```

File: docs/migration.md [hunk 2]
```diff
@@ -270,8 +273,48 @@ or [Local Configuration File](#local-configuration-file) for migration details.
...
+
+#### Name
+
+Define the name after the slash `/` in the receiver name.
+
+To set `_sourceName`, use [resourceprocessor][resourceprocessor]
+or set it in [sumologicexporter][sumologicexporter].
+
+For example, the following snippet configures the name for [Filelog Receiver][filelogreceiver] instnace as `my example name`:
```
Comment (aboguszewski-sumo) [inline docs/migration.md:287]:
```suggestion
For example, the following snippet configures the name for [Filelog Receiver][filelogreceiver] instance as `my example name`:
```

File: docs/migration.md [hunk 3]
```diff
@@ -270,8 +273,48 @@ or [Local Configuration File](#local-configuration-file) for migration details.
...
+exporters:
+  sumologic:
+    source_name: my example name
+```
+
+#### Description
+
+A description can be added as a comment just above the receiver name.
+
+For example, the following snippet configures the description [Filelog Receiver][filelogreceiver] instnace as `All my example logs`:
```
Comment (aboguszewski-sumo) [inline docs/migration.md:302]:
```suggestion
For example, the following snippet configures the description for [Filelog Receiver][filelogreceiver] instance as `All my example logs`:
```

File: docs/migration.md [hunk 4]
```diff
@@ -353,37 +396,11 @@ service:
 
 #### Name
 
-Define the name after the slash `/` in the receiver name.
```
Comment (rnishtala-sumo) [inline docs/migration.md:356]: I guess we must do something similar for all the other sources? Could we also move `Source Category` and `Fields` to a common section?

Comment (sumo-drosiek) [reply docs/migration.md:356]: Sure, i just won't be able to deliver it Today :sweat_smile:

Comment (sumo-drosiek) [reply docs/migration.md:356]: Wanted to show the direction we have been talking about Yesterday

**Review by rnishtala-sumo — APPROVED** [review]

**Pull request merged**